__tests__/app.test.js

```diff
@@ -51,4 +51,41 @@ describe("GET/api/", () => {
             expect(body.msg).toBe('Route not found');
           });
       });
+  });
+  describe('GET/api/articles/:article_id', () => {
+    it('200: responds with requested article', () => {
+        const articleId = 1;
+        return request(app)
+        .get(`/api/articles/${articleId}`)
+        .expect(200)
+        .then((res) => {
+            const { body } = res;
+            const { article } = body;
+            expect(article).toHaveProperty('author', expect.any(String));
+            expect(article).toHaveProperty('title', expect.any(String));
+            expect(article).toHaveProperty('article_id', articleId);
+            expect(article).toHaveProperty('body', expect.any(String));
+            expect(article).toHaveProperty('topic', expect.any(String));
+            expect(article).toHaveProperty('created_at', expect.any(String));
+            expect(article).toHaveProperty('votes', expect.any(Number));
+            expect(article).toHaveProperty('article_img_url', expect.any(String));
+        });
```

Comment:
This is not the end of the world, you are asserting specifically for article id 1 - which is important!
You could also assert that other properties are specific too!